cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -2252,19 +2252,33 @@ def setUp(self):
         XBlockStudioConfiguration.objects.create(name='discussion', enabled=True, support_level="ps")
         XBlockStudioConfiguration.objects.create(name='problem', enabled=True, support_level="us")
         XBlockStudioConfiguration.objects.create(name='video', enabled=True, support_level="us")
-        # XBlock masquerading as a problem
         XBlockStudioConfiguration.objects.create(name='openassessment', enabled=True, support_level="us")
+        # XBlock masquerading as a problem
         XBlockStudioConfiguration.objects.create(name='drag-and-drop-v2', enabled=True, support_level="fs")
+        XBlockStudioConfiguration.objects.create(name='staffgradedxblock', enabled=True, support_level="us")
```

Comment:
@farhaanbukhsh Why are you adding `staffgradedxblock` here?

Reply:
Since we moved open assessment out, I wanted `problems` to still have at least two components so instead of `drag-and-drop-v2` and `openassessment`, I introduced `staffgradedxblock` to check the template. Should I remove it?

Reply:
Ah, no problem with this. I haven't noticed until I've read the entire file. 
We're good to keep this here.